src/main/webapp/repo-bootstrap/global/blueprints/website_editorial/config/studio/administration/samples/sample-aws.xml

```diff
@@ -54,4 +54,17 @@ For every profile you need to specify at least:
             </output>
         </outputs>
     </profile>
+    <profile>
+        <id>mediaconvert-default</id>
```

Comment:
The documentation of the file inside <!-- --> needs to be updated too, since a new type of profile was introduced.